ac1/_sources/introduction_to_visualizations/summary.rst

```diff
@@ -0,0 +1,10 @@
+.. Copyright (C)  Google, Runestone Interactive LLC
+   This work is licensed under the Creative Commons Attribution-ShareAlike 4.0
+   International License. To view a copy of this license, visit
+   http://creativecommons.org/licenses/by-sa/4.0/.
+
+.. image:: figures/summary.png
+   :align: center
+   :alt: A line graph comparing number of likes over time of two posts.
+   
+.. _ Location of summary document: shorturl.at/nswJ5
```

Comment:
Quick question: What does the _ before "Location of summary document" do?

Reply:
@YBirhane

Reply:
It just has the google docs location of all the summaries we made (and we made the link publicly accessible).

Reply:
I removed the underscore!